### PR TITLE
docs: Use capital letters for CSV and JSON acronyms

### PR DESCRIPTION
**Pull request by nicokosi**
Capital letters matter! 😉 
Screenshot 2023-03-15 at 06 54 22

See https://github.com/nushell/nushell.github.io/pull/827/files that was created before this one.

### CODE REVIEW

File: crates/nu-command/src/dataframe/eager/open.rs [hunk 1]
```diff
@@ -22,7 +22,7 @@ impl Command for OpenDataFrame {
     }
 
     fn usage(&self) -> &str {
-        "Opens csv, json, arrow, or parquet file to create dataframe."
+        "Opens CSV, json, arrow, or parquet file to create dataframe."
```
Comment (jntrnr) [inline crates/nu-command/src/dataframe/eager/open.rs:25]: If we do this, JSON is also all caps

Comment (nicokosi) [reply crates/nu-command/src/dataframe/eager/open.rs:25]: I agree, but I though about doing this in a seperate pull request.
What do you think?

Comment (fdncred) [reply crates/nu-command/src/dataframe/eager/open.rs:25]: I think this is a can-of-worms type thing. You'll be searching the entire repo for csv, json, yaml, yml, eml, ics, nuon, ods, parquet, ssv, toml, tsv, vcf, xlsx, xml, etc. I'm personally fine with leaving them all lowercase.

Comment (rgwood) [reply crates/nu-command/src/dataframe/eager/open.rs:25]: > I though about doing this in a separate pull request.

That's too granular IMO; if you have multiple small spelling fixes I would prefer 1 pull request.

Comment (nicokosi) [reply crates/nu-command/src/dataframe/eager/open.rs:25]: OK, I'll do further "acronym fixes" in this pull request. I mark it as a draft.

**Comment by nicokosi** [comment]
I have checked for "acronym issues" in the documentation via:
```sh
ag --case-sensitive csv
ag --case-sensitive yaml
ag --case-sensitive toml
ag --case-sensitive xml
```
It's now correct for me! 😅

File: crates/nu-command/src/dataframe/eager/open.rs (hunk 1 above)
Comment (nicokosi) [reply crates/nu-command/src/dataframe/eager/open.rs]: The PR has now fixes for both CSV (ceef0ba067cad437de2a65e71b5ff5ba4649583a) and JSON (331feb2d9ac6bc4b83d217df6b40e3bd32259ea3) acronyms. 🤓

**Pull request merged**